PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## What does this PR do?
+
+<!-- MANDATORY -->
+
+<!-- explain the changes in your PR, algorithms, design, architecture -->
+
+## Why is this change important?
+
+<!-- MANDATORY -->
+
+<!-- explain the motivation behind your PR -->
+
+## How to test this PR locally?
+
+<!-- commands to run the tests or instructions to test the changes-->
+
+## Author's checklist
+
+<!-- additional notes for reviewiers -->
+
+## Related issues
+
+<!--
+Closes #234
```

Comment:
I think we should remove the number to not mention that issue by accident a lot.

Reply:
I am not sure I understand your issue with this. Could you provide more information? This section is commented out, so Github will not close anything even if you leave this line in your PR description.

Reply:
Some PR's authors may not change the default value because it is in a comment section, so the issue 234 will referenced a lot of time for no reason.